integration-tests/tests/lock-context.test.ts

```diff
@@ -0,0 +1,93 @@
+import 'dotenv/config'
+import { describe, expect, test } from 'vitest'
+
+import {
+  type CastAs,
+  decrypt,
+  encrypt,
+  newClient,
+} from '@cipherstash/protect-ffi'
+
+const encryptConfig = {
+  v: 1,
+  tables: {
+    users: {
+      email: {
+        cast_as: 'text' as CastAs, // FIXME: do we need the as ?
+        indexes: {
+          ore: {},
+          match: {},
+          unique: {},
+        },
+      },
+      score: {
+        cast_as: 'double' as CastAs,
+        indexes: {}, // TODO: add ore index options here when we support them
```

Comment:
What's preventing using ORE indexes with this change? If they aren't supported, can we track that separately instead of using a TODO, please?